Dokument/Dateien/Religion.tex

```diff
@@ -30,6 +30,175 @@ \section{Götter}
+Bouda ist die schaffende Gottheit des Pantheons. Nach ihrem Willen erwachsen neue Pflanzen in jedem Frühling, um den Hunger aller zu stillen. Nach ihrem Wilen wir das Band 
+der Ehe geformt, das Mann und Frau in Harmonie miteinander verbindet. Bouda akzeptiert alle als ihre Kinder und schenkt die Fähigkeit, zu vergeben und zu versöhnen.\\
+\begin{itemize}
+	\item Aussehen 
+	\begin{itemize}
+		\item ``Mutterfigur''
+		\item mittleres Alter 
+		\item kräftige Figur
+		\item alltägliche Kleidung 
+	\end{itemize}
+	\item Aspekte
+	\begin{itemize}
+		\item Familie (Ehe und Elternschaft)
+		\item Ernte und Wachstum
+		\item Heilung
+	\end{itemize}
+	\item Tugenden Boudas
+	\begin{itemize}
+		\item Gnade und Demut
+		\item Offenheit und Akzeptanz
+		\item nichterotische Liebe
+	\end{itemize}
+\end{itemize}
+\textit{Weit bekannter Mythos Boudas}:\\
+Vor Urzeiten, als die Welt noch jünger war, lagen die Reiche der Menschen beständig im Streit miteinander. Päkte wurden geschmiedet und gebrochen, ein Herrscher folgte auf den 
+nächsten und keiner vermochte es, andauernden Frieden zu finden. Doch manchmal schmiedete das Schicksal seltsame Bande: es begab sich nämlich, dass die Thronfolger zweier 
+verfeindeter Reiche in demselben Wald jagten und von einem Sturm von ihren jeweiligen Wegen abgebracht und in der 
+Hütte einer alten Frau zusammengebracht wurden. Und als die beiden sich der Anziehung bewusst wurden, die zwischen ihnen zu erblühen begann, baten sie die alte Frau, ihre
+Vermählung vor Bouda zu bezeugen, auf dass selbst ihre Väter sie nicht mehr trennen könnten. Und die alte Frau schenkte ihnen zur Mitgift zwei Armreife, die das Band ihrer Vereinigung 
+darstellen sollten. Und als die beiden Thronfolger heimkehrten, kam Friedfertigkeit über jeden, der die Reife der beiden sah. Und zum ersten Mal seit langem herrschte Frieden 
+zwischen den Nationen.
+
+\subsection{\textbf{Erlin} - Gott des Ausgleichs und der Gerechtigkeit}
+Erlin lehrt die Menschen, die Bindungen zu ihren Mitmenschen ausgewogen zu halten. Sowohl Respektlosigkeit als auch übermäßige Ehrerbietung versäuern das Verhältnis 
+zu den Mitmenschen und muss daher vermieden werden. Dazu gehört sich an die Gesetze und Regln der Gesellschaft zu halten.\\
+\begin{itemize}
+	\item Aussehen 
+	\begin{itemize}
+		\item alter Mann 
+		\item hochgewachsen und hager
+		\item in makelloser Robe, aufrechte Haltung
+		\item Richthammer am Gürtel
+	\end{itemize}
+	\item Aspekte
+	\begin{itemize}
+		\item Gesetzgebung und Rechtsprechung
+		\item Handel
+		\item Weises Vorgehen \footnote{Leichte Überschneidung mit Faelans Aspekt}
+	\end{itemize}
+	\item Tugenden Erlins
+	\begin{itemize}
+		\item Disziplin und Respekt
+		\item stoisch und gelassen
+		\item streng mit sich selbst und Anderen
+	\end{itemize}
+\end{itemize}
+\textit{Weit bekannter Mythos Erlins}:\\
+<Erlin fällt eine Richtspruch über ...>
+
+\subsection{\textbf{Aina} - Göttin der Lebensfreude und Liebe}
```

Comment:
Bei dieser Göttin könnte man noch einen Part hinzufügen, der darauf eingeht, dass sie diejenige ist, die Hoffnung schenkt in Bezug auf Lebensfreude. Also auch wenn Verwandte gestorben sind, so darf man sich davon nicht unterkriegen lassen. So was und ähnliches zB. Also dass sie nicht nur eine reine Party-Göttin ist, sondern auch diesen etwas ernsteren und recht wichtigen Hintergrund hat